release-tools/src/main/java/org/springframework/data/release/cli/IssueTrackerCommands.java

```diff
@@ -36,12 +38,14 @@
 
 /**
  * @author Oliver Gierke
+ * @author Mark Paluch
  */
 @CliComponent
 public class IssueTrackerCommands implements CommandMarker {
 
 	private final PluginRegistry<IssueTracker, Project> tracker;
 	private final JiraConnector jira;
+	private final GitHubIssueConnector gitHub;
```

Comment:
So merging the commands and iterating over elements in `IssueTrackerCommands`? What about the `jira` and `...Evict` methods?